src/components/partner/partner.service.ts

```diff
@@ -178,6 +185,23 @@ export class PartnerService {
     };
   }
 
+  async listLanguages(
+    partner: Partner,
+    input: LanguageListInput,
+    session: Session,
+  ): Promise<SecuredLanguageList> {
+    const languageListOutput = await this.languageService.list(
+      { ...input, filter: { ...input.filter, partnerId: partner.id } },
+      session,
+    );
+    return {
+      ...languageListOutput,
+      canRead: true,
+      // TODO - rethink through the privileges for this
+      canCreate: this.privileges.for(session, IProject).can('create'),
```

Comment:
Struggling a bit here @CarsonF - would this be the Project or the Engagement or would it actually be the language?

Reply:
It's a bit meaningless from this perspective though, since Partner's don't own a relationship to Languages.
For a new language to show up in this list, a project, partnership, engagement, and maybe a new language(?) would have to be created.
I'd be fine just hardcoding to `false`, with some kind of "non-owned list" comment.